bundle-core/src/main/java/net/discdd/utils/ADUUtils.java

```diff
@@ -58,7 +60,7 @@ public static List<ADU> readADUs(File aduDirectory) {
             String appId = appSubDirectory.getName();
             for (final File aduFile : appSubDirectory.listFiles(file -> !file.isHidden())) {
                 String aduFileName = aduFile.getName();
-                long aduId = Long.valueOf(aduFileName.split("-")[1]);
+                long aduId = Long.parseLong((aduFileName.split("-")[1]).replace(".adu",""));
```

Comment:
careful this might have changed. we are trying to move to the adu files just named with their aduid. no - and no .adu extension.